res/layout-v14/generic_explanation.xml

```diff
@@ -49,15 +49,17 @@
         android:orientation="horizontal" >
 
         <!-- 'OK' / 'CANCEL' BUTTONS CHANGE THEIR ORDER FROM ANDROID 4.0 ; THANKS, GOOGLE -->
-        <Button
+        <android.support.v7.widget.AppCompatButton
```

Comment:
Now v14 is the minimum valid API; there is no need to have two separece generic_explanation.xml files. This one should replace the one in res/layout, then the folder layout-v14 may be removed.